flipperdevices/flipperzero-firmware
# [FL-1913, FL-1963] SubGhz: save raw signal, add came atomo decoder

# What's new

- [FL-1913] SubGhz: reception, recording, graphical interface, playback, RAW signal
- [FL-1963] SubGhz: fix freeze while saving signal
- Canvas: add font rotation
- File_Worker: getting the name of a new file with an index
- SubGhz: add decode came atomo

# Verification 

- compile and download
- launch the SubGhz application, accept the CAME Atomo parcel, see the result (requires an additional file on the CD card)
- Open the RAW signal recording application, record any signal at the desired frequency and then play it back from the saved ones

# Checklist (do not modify)



[FL-1913]: https://flipperzero.atlassian.net/browse/FL-1913?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FL-1963]: https://flipperzero.atlassian.net/browse/FL-1963?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
